crates/ty_python_semantic/resources/mdtest/typed_dict.md

```diff
@@ -25,12 +25,21 @@ alice: Person = {"name": "Alice", "age": 30}
 reveal_type(alice["name"])  # revealed: Unknown
 # TODO: this should be `int | None`
 reveal_type(alice["age"])  # revealed: Unknown
+
+# TODO: this should reveal `Unknown`, and it should emit an error
+reveal_type(alice["non_existing"])  # revealed: Unknown
```

Comment:
So.. all of these still don't work, because the type of `alice` here is simply `dict[Unknown, Unknown]`. See below for the real tests for this feature.